<launch>
	<arg name="scene" default="1"/>
	<arg name="save" default="true"/>

	<!--<rosparam command="load" file="$(find pr2_robot)/config/pick_list_3.yaml"/>-->
	<node pkg="pr2_robot" type="pr2_pick_place_client.py" name="pr2_pick_place_client" output="screen">
		<param name="scene" value="$(arg scene)"/>
		<param name="save_yaml" value="$(arg save)"/>
		<param name="yaml_file" value="/tmp/output_$(arg scene).yaml"/>
	</node>
</launch>
